src/sql/workbench/services/connection/browser/passwordChangeDialog.ts

```diff
@@ -143,6 +141,8 @@ export class PasswordChangeDialog extends Modal {
 			return Promise.reject(new Error(passwordChangeResult.errorMessage));
 		}
 		connection.options['password'] = newPassword;
+		// Only one type is valid, the default one.
+		let params: INewConnectionParams = { connectionType: ConnectionType.default }
 		await this.connectionDialogService.callDefaultOnConnect(connection, params);
```

Comment:
is this going to reuse the connection dialog that was opened? please attach gif demonstrating it.

Reply:
No, it will not, it connects the profile directly after entering similar to how it is done in SSMS (this was by request), although the changed password will persist in the profile in connection explorer.